hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This adds support for partially sorting examples which cannot be fully sorted.
```

Comment:
I think this needs either more detail or less. As written this doesn't make much sense to anyone who doesn't already know what it does - "This improves the shrinker in some cases" would be fine, as would this version with an example.

Reply:
Ack. I added an example, since I think a little more detail is more useful than having a lot of identical change entries.